### ISSUE TITLE
[Outreachy Task Submission] Do not display `contact email address` when empty.

### ISSUE THREAD

**Issue by nimishmedatwal** [ushahidi/platform#4866]
Problem with accordance to accessibility guidelines: 

- The element lacks visible text that screen readers can interpret, thereby hindering accessibility for visually impaired users.
- It is included in the tab order but lacks accessible text, which means keyboard users may not understand its purpose or function.
-  Empty <a> tag, providing no meaningful content or interaction for users.
```html
 <a _ngcontent-cll-c66="" class="company-info__email" href="mailto:" data-qa="contact-email"></a> 
```

**Proposed solution**

- Do not display `contact email address when it's empty.
When empty
![image](https://github.com/ushahidi/platform/assets/91622060/375d3c20-55ee-46c1-8045-5ac71db25cf5)
When not empty
![image](https://github.com/ushahidi/platform/assets/91622060/d2a5eaf1-45c4-44d9-ab7e-2e3a5fc9f89f)

- Accessibility Attributes: Ensure that the <a> tag has appropriate accessibility attributes to make it accessible to users of assistive technologies, such as screen readers. This includes adding aria-label or title attributes to provide additional context for the link.

**Additional context**
![image](https://github.com/ushahidi/platform/assets/91622060/2444953b-fda3-483e-b243-67ad40bdb843)
